index.js

```diff
@@ -0,0 +1,36 @@
+const upperCase = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+const lowerCase = 'abcdefghijklmnopqrstuvwxyz'
```

Comment:
These should live inside the function, we don't declare variables outside of functions like this.